zorrom/mrom.py

```diff
@@ -259,7 +304,7 @@ def run(self, rotate=None, flipx=False, flipy=False):
 
             def next_byte():
```

Comment:
If we move to a word architecture, this should become next_word() and a next_byte() that depends on next_word(). I think that would simplify the code a lot